src/lib/Form/Type/Search/SearchType.php

```diff
@@ -28,16 +29,17 @@ public function __construct(AbstractType $baseType, TranslatorInterface $transla
     }
 
     /**
-     * @param \Symfony\Component\Form\FormBuilderInterface $builder
-     * @param array $options
-     *
      * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $this->baseType->buildForm($builder, $options);
 
         $builder
+            ->remove('search_in_users');
```

Comment:
I'm having trouble locating this field, from where does it come from and why do you need to drop it?